userguide/sharing.rst

```diff
@@ -2620,3 +2476,142 @@ Return to
 size of the file.
+
+
+.. index:: Time Machine
+.. _Creating Authenticated and Time Machine Shares:
+
+Creating Authenticated and Time Machine Shares
+----------------------------------------------
+
+macOS includes the
+`Time Machine <https://support.apple.com/en-us/HT201250>`__ feature
+which can be used to perform automatic back ups. %brand% supports
+Time Machine back ups for both :ref:`AFP <Apple (AFP) Shares>` and
+:ref:`SMB <Windows (SMB) Shares>` shares. The process for creating an
+authenticated share for a user is the same as creating a Time Machine
+share for that user.
+
+Before creating a Time Machine or authenticated share, go to
+:menuselection:`Storage --> Pools` to make a dataset for the share.
+For more information about dataset creation, refer to
+:ref:`Adding Datasets`.
+
+After creating the dataset for the guest share, go to
+:menuselection:`Storage --> Pools`,
+click the |ui-options| button for the dataset, then
+:guilabel:`Edit Permissions`.
+
+Enter the following information as shown in
+:numref:`Figure %s <creating_an_authenticated_share_fig>`.
+
+#. **ACL Type:** Select :guilabel:`Mac`.
+
+#. **User:** Use the drop-down to select the desired user account.
+   If the user does not yet exist on the %brand% system, go to
+   :menuselection:`Accounts --> Users`
+   to create one. Refer to
+   :ref:`users <Users>` for more information about creating a user.
+
+#. **Group:** Use the drop-down to select the desired group name.
+   If the group does not yet exist on the %brand% system, go to
+   :menuselection:`Accounts --> Groups`
+   to create one. Refer to
+   :ref:`groups <Groups>` for more information about creating a group.
+
+#. Click :guilabel:`SAVE`.
+
+
+To create an authenticated or Time Machine share:
+
+#. Go to :menuselection:`Sharing --> Apple (AFP) Shares`
+   or
+   :menuselection:`Sharing --> Windows (SMB) Shares`
+   and click |ui-add|.
+
+#. :guilabel:`Browse` to the dataset created for
+   the share.
+
+#. When creating a Time Machine share, enable the
+   :guilabel:`Time Machine` option.
+
+#. Fill out the other required fields.
+
+#. Click :guilabel:`SAVE`.
+
+To configure multiple authenticated or Time Machine shares, repeat
+this process for each user.
+:numref:`Figure %s <creating_an_authenticated_share_fig>` shows
+creating a Time Machine Share in
+:menuselection:`Sharing --> Apple (AFP) Shares`.
+
+.. _creating_an_authenticated_share_fig:
+
+.. figure:: images/sharing-apple-afp-add.png
+
+   Creating an Authenticated or Time Machine Share
+
+
+At this point, it may be desirable to configure a quota for each Time
+Machine share, to restrict backups from using all of the available
+space on the %brand% system. The first time Time Machine makes a
+backup, it will create a full backup after waiting two minutes. It
+will then create a one hour incremental backup for the next 24 hours,
+and then one backup each day, each week and each month.
```

Comment:
```suggestion
After that, one backup is created each day, each week, and each month.
```

Reply:
This is still kind of misleading, since all of these backups are updated.  This kind of implies that there are only hourly backups of the first day.  Really, all of these backups are updated on their own schedule.  In other words, hourly backups are updated hourly, daily are updated daily, and so on.